WindowsServerDocs/remote/remote-access/directaccess/single-server-wizard/da-basic-plan-s1-infrastructure.md

```diff
@@ -110,6 +110,17 @@ In a DirectAccess deployment, DNS is required for the following:
 
     2.  If the corporate network is IPv6-based, the default address is the IPv6 address of DNS servers in the corporate network.
 
+> [!NOTE]
+> Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy Table (NRPT).
+> If DNS registration is needed, for example **Manage Out**, it can be explicitly enabled with this registry key on the client:
+>
+> Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`
+> Type: `DWORD`
+> Value name: `DisableNRPTForAdapterRegistration`
+> Values:
+> `1` - DNS Registration disabled (default since the Windows 10 May 2020 Update)
+> `0` - DNS Registration enabled
```

Comment:
Repeating what I noted in your PR #5207 : We need to add HTML <br/> to enable actual line breaks in a MarkDown Note blob (otherwise, all these lines will show up as one contiguous line).
```suggestion
> Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`<br/>
> Type: `DWORD`<br/>
> Value name: `DisableNRPTForAdapterRegistration`<br/>
> Values:<br/>
> `1` - DNS Registration disabled (default since the Windows 10 May 2020 Update)<br/>
> `0` - DNS Registration enabled
```